Geometry/HGCalCommonData/src/HGCalDDDConstants.cc

```diff
@@ -884,6 +885,7 @@ std::vector<int> HGCalDDDConstants::numberCells(int lay, bool reco) const {
   if (i >= 0) {
     if (waferHexagon6()) {
       for (unsigned int k = 0; k < hgpar_->waferTypeT_.size(); ++k) {
+        std::cout << "SIZE WAFERTYPET_:  " << hgpar_->waferTypeT_.size() << std::endl;
```

Comment:
`edm::LogInfo` ?

Reply:
`cout` is not allowed in production code.

Reply:
This was for debugging purposes, and I forgot to remove it. Same for all other ```cout```s.